sam/onyx/hw_nodes/pass_through_node.py

```diff
@@ -151,7 +144,7 @@ def connect(self, other, edge, kwargs=None):
         return new_conns
 
     def get_connection_from_tensor(self, tensor):
-        print(self.tensor_to_conn)
+        # print(self.tensor_to_conn)
```

Comment:
some of these print stmt are commented out instead of being deleted, is there a reason why we wanna keep these?

Reply:
I kept some if I thought it would make debugging easier later